neptune.yaml

```diff
@@ -52,13 +49,18 @@ parameters:
   image_channels: 3
 
 # U-Net parameters
-  nr_unet_outputs: 4
+  unet_output_channels: 2
```

Comment:
@taraspiotr is this for the solution-0 ?
Also I think that unet_mask_... is more descriptive than unet_simple... 
Can we drop one of those (unet...) and keep two configs like neptune_solution_0.yaml neptune_solution_1.yaml or going one step further have `cfgs` dir and in there have solution_0.yaml solution_1_masks.yaml solution_1_masks_with_borders.yaml ?

Reply:
@taraspiotr Please don't remove `nr_unet_outputs` parameter. It is needed when you build UNet from scratch to determine number of output tensors from UNet.